octopusdeploy_framework/schemas/tenant.go

```diff
@@ -132,8 +144,13 @@ func (t TenantSchema) GetResourceSchema() resourceSchema.Schema {
 			},
 			"description": GetDescriptionResourceSchema("tenant"),
 			"id":          GetIdResourceSchema(),
-			"name":        GetNameResourceSchema(true),
-			"space_id":    GetSpaceIdResourceSchema("tenant"),
+			"is_disabled": resourceSchema.BoolAttribute{
```

Comment:
Looking at the project resource schema the is_disabled schema has a `boolplanmodifier.UseStateForUnknown()` I think we need it here

> [UseStateForUnknown()](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-framework/resource/schema/boolplanmodifier#UseStateForUnknown): Copies a known prior state value into the planned value. Use this when it is known that an unconfigured value will remain the same after a resource update.